files/files.go

```diff
@@ -114,6 +114,36 @@ func (c *Content) Sys() interface{} {
 	return nil
 }
 
+type EmptyFolder struct {
+	Path  string      `yaml:"path,omitempty"`
+	Owner string      `yaml:"owner,omitempty"`
+	Group string      `yaml:"group"`
+	Mode  os.FileMode `yaml:"mode,omitempty"`
+	MTime time.Time   `yaml:"mtime,omitempty"`
```

Comment:
missing the jsonschema tags

Reply:
I think maybe it is a good idea to keep the `EmptyFolder` consistent with `Content`/`ContentInfo`. 
Are jsonschema tags necessary?

Reply:
I would try to make this Just part of the content structure with an `emptyfolder` type. And leave the current empty folder structure alone for backwards compatibility. We can then depreciate the empty folder structure.

Reply:
we could also write a custom decoder for this type, so the old format (string only) still works...